package.json

```diff
@@ -22,7 +22,7 @@
     "tap": "^10.0.1"
   },
   "engines": {
-    "node": "7"
```

Comment:
Please add a reason for updating the engine version? Are you using features that are not available earlier?

Also, this should be a separate commit.